tests/Hook/Storage/CallableOperationTest.php

```
@@ -0,0 +1,86 @@
+        $operation = new CallableOperation(fn () => ['key' => 'value', 'number' => 123]);
+
+        $result = $operation->execute();
+
+        self::assertIsArray($result);
+        self::assertSame(['key' => 'value', 'number' => 123], $result);
+    }
+
+    public function testExecutePreservesExceptions(): void
+    {
+        $operation = new CallableOperation(function () {
+            throw new \RuntimeException('Test exception');
```

PHPMD (2.15.0)

Missing class import via use statement (line '78', column '23'). (undefined)

(MissingImport)